common/app/common/configuration.scala

```diff
@@ -440,10 +440,6 @@ class GuardianConfiguration extends Logging {
     lazy val expiredPaidContentUrl = s"${site.host}/info/2015/feb/06/paid-content-removal-policy"
   }
 
-  object open {
-    lazy val ctaApiRoot = configuration.getMandatoryStringProperty("open.cta.apiRoot")
```

Comment:
The config files lives on S3. Here are the manual step to update it https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/12-Update-configuration-in-s3.md

The DEVINFRA file is only used when running the app on Teamcity. It set to empty or fake values some mandatory config parameters.